### ISSUE TITLE
Is it possible to run more then one TFLite models at the same time on a Flutter App

### ISSUE THREAD

**Issue by chaoyifei** [tensorflow/flutter-tflite#210]
is  there  any  way to use yolov5 and  osnet tflite models  at same time?

**Comment by DevNim98** [comment]
Yes it is totally posible, you just need to instantiate two interpreters, each one with their respective model. Then parse the output of each one according to their output tensor. The only thing that may prohibit you of using more than one model, is the available RAM on your device, since allocating the model and the output result requires a specific amount on RAM.

**Comment by prateeks-myelin** [comment]
Hi @DevNim98, is it possible to make 1 tflite model from 2 tflite models where the output of 1 model becomes the input of another? i do not want to call 2 tflite models as i am resource constrained and want to optimize 2 models working in tandem. both models are trained for different use cases.
